samples/msal-browser-samples/VanillaJSTestApp2.0/app/customizable-e2e-test/test/browserAAD.spec.ts

```diff
@@ -25,49 +24,72 @@ async function verifyTokenStore(BrowserCache: BrowserCacheUtils, scopes: string[
 }
 
 
-describe("AAD-PPE Tests", () => {
+describe("AAD Tests", () => {
     let browser: puppeteer.Browser;
     let context: puppeteer.BrowserContext;
     let page: puppeteer.Page;
     let BrowserCache: BrowserCacheUtils;
+    let clientID: string;
+    let authority: string;
     let username = "";
     let accountPwd = "";
+    let msalConfig: Configuration;
+    let request: any;
 
     beforeAll(async () => {
         createFolder(SCREENSHOT_BASE_FOLDER_NAME);
         browser = await getBrowser();
         sampleHomeUrl = getHomeUrl();
 
         const labApiParams: LabApiQueryParams = {
-            azureEnvironment: AzureEnvironments.PPE,
-            appType: AppTypes.CLOUD
+            azureEnvironment: AzureEnvironments.CLOUD,
+            appType: AppTypes.CLOUD,
+            userType: UserTypes.CLOUD,
+            appPlatform: AppPlatforms.SPA,
         };
 
         const labClient = new LabClient();
         const envResponse = await labClient.getVarsByCloudEnvironment(labApiParams);
 
         [username, accountPwd] = await setupCredentials(envResponse[0], labClient);
+        [clientID, , authority] = await retrieveAppConfiguration(envResponse[0], labClient, false);
```

Comment:
Wasn't the while idea of a separate json file is to create the flexibility to change the config as needed in unit tests? Why is this code refactored now? cc @hectormmg  who made the initial changes to review this.

Reply:
Well I think the util `retrieveAppConfiguration` was added more recently (perhaps it wasn't quite needed until cca testing), but now that we have it, its safer to not depend on hardcoded ids and just retrieve as much as possible from the lab at runtime (there's still a json file, its just being populated at runtime)